HULKs/hulk
# Use Footbumpers to spawn unseen obstacles

## Introduced Changes

The robot makes use of the foot bumper to spawn obstacles it recognizes by stepping into it.

Fixes #502 , #510 

## ToDo / Known Issues

Needs to be tested in test games, to see potential issues

## Ideas for Next Iterations (Not This PR)
- Think about having different life times for sonar obstacles and footbumper obstacles
- Merge counter for center obstacles

## How to Test

Has to be enabled in config
Needs to be tested when you let a robot walk and observe in twix if it spawns no existing obstacles.
Tape one fottbumper to force constant triggering, and see if it does not spawn obstacles in that case.
